dockerfiles/Dockerfile.build

```diff
@@ -0,0 +1,14 @@
+FROM openjdk:11
+LABEL maintainer="Digirati <opensource@digirati.com>"
+
+USER root
+
+RUN apt-get clean && apt-get update && apt-get -y upgrade \
+    && apt-get install -y python python-pip software-properties-common lsb-release apt-transport-https \
```

Comment:
Let's target specific versions here. We can find the versions currently available using `apt-cache madison <package-name>`.